pcweb/components/navbar.py

```diff
@@ -296,6 +296,12 @@ def navbar(sidebar: rx.Component = None) -> rx.Component:
                         style=styles.NAV_TEXT_STYLE,
                         display=["none", "none", "none", "flex", "flex", "flex"],
                     ),
+                    rx.link(
```

Comment:
Should this go under the drop down menu, or you think it should be on the main level?

Reply:
Main level